boards/arm/sam_v71_xult/sam_v71_xult_defconfig

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_SAM=y
+CONFIG_SOC_SERIES_SAMV71=y
+CONFIG_SOC_PART_NUMBER_SAMV71Q21=y
+CONFIG_SOC_ATMEL_SAMV71_EXT_MAINCK=y
+CONFIG_SOC_ATMEL_SAMV71_PLLA_MULA=24
+CONFIG_SOC_ATMEL_SAMV71_PLLA_DIVA=1
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=300000000
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_USART_SAM=y
+CONFIG_USART_SAM_PORT_1=y
+CONFIG_BOARD_SAM_V71_XPLAINED=y
+CONFIG_BUILD_OUTPUT_HEX=y
```

Comment:
This has been added before as a workaround. In a meantime the build system has been modified to always build hex. I propose to remove it. As a long term solution we should fix openocd scripts to reliably flash bin files.